### PR TITLE
Fixing shading

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/BigQueryInputSplit.java [hunk 1]
```diff
@@ -44,9 +47,6 @@
 import org.apache.hadoop.mapred.JobConf;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import shaded.hivebqcon.com.google.cloud.bigquery.connector.common.*;
-import shaded.hivebqcon.com.google.common.annotations.VisibleForTesting;
-import shaded.hivebqcon.com.google.common.collect.ImmutableList;
```
Comment (yigress) [inline connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/BigQueryInputSplit.java:49]: don't know if the issue of java.lang.NoSuchFieldError: VIRTUAL_COLUMN_NAMES is relate to the change here.

**Review by functicons — APPROVED** [review]
LGTM

**Pull request merged**